### PR TITLE
dxChart - Fix axes disposing when updating panes (T1042940)

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: testing/tests/DevExpress.viz.charts/chart.integration.tests.js [hunk 1]
```diff
@@ -2016,6 +2016,83 @@ QUnit.test('Chart with large scale break', function(assert) {
...
+            'val': 44,
+            'series': 'Asia | Count',
+            'arg': '2014'
+        }],
+        seriesTemplate: {
+            nameField: 'series'
+        }
+    });
+
+    assert.ok(chart);
```
Comment (alexanderPolosatov) [inline testing/tests/DevExpress.viz.charts/chart.integration.tests.js]: may be remove this assertion?

File: js/viz/chart.js [hunk 2]
```diff
@@ -641,7 +641,12 @@ const dxChart = AdvancedChart.inherit({
             if(!axis.pane) {
                 axis.setPane(that.defaultPane);
             }
-            return doesPaneExist(that.panes, axis.pane);
+            const paneExists = doesPaneExist(that.panes, axis.pane);
+            if(!paneExists) {
+                axis.dispose();
+                axis = null;
```
Comment (alexanderPolosatov) [inline js/viz/chart.js:647]: axis = null;
I think it's extra in filter function. I'm right?

Comment (dxBeardedBear) [reply js/viz/chart.js:647]: Not quite. It is better not to rely on GC and reset the link

File: testing/tests/DevExpress.viz.charts/chart.integration.tests.js (hunk 1 above)
Comment (dxBeardedBear) [reply testing/tests/DevExpress.viz.charts/chart.integration.tests.js]: done

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**